apps/commercetools/src/AppConfig/AppConfig.tsx

```diff
@@ -0,0 +1,251 @@
+import * as React from 'react';
+
+import get from 'lodash/get';
+import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
+import {
+  Heading,
+  Paragraph,
+  Typography,
+  TextField,
+  Form
+} from '@contentful/forma-36-react-components';
+import tokens from '@contentful/forma-36-tokens';
+import { css } from 'emotion';
+
+import FieldSelector from './FieldSelector';
+
+import { toInputParameters, toAppParameters } from './parameters';
+
+import {
+  getCompatibleFields,
+  editorInterfacesToSelectedFields,
+  selectedFieldsToTargetState,
+  EditorInterface,
+  ContentType,
+  CompatibleFields,
+  FieldsConfig
+} from './fields';
+
+import { validateParameters } from './validateParameters';
+
+import { Hash } from '../interfaces';
+import logo from '../logo.svg';
+import descriptor from '../../extension.json';
```

Comment:
In a long run we should drop use of `extension.json` completely (we're not doing extensions here).

You import it only to grab parameter definitions. Just extract them to a regular module and drop dependency on `extension.json`

Reply:
dropped now